packages/react-components/src/components/ControlBar.tsx

```diff
@@ -54,23 +54,23 @@ export interface ControlBarProps {
 export const ControlBar = (props: ControlBarProps): JSX.Element => {
   const { styles, layout } = props;
   const theme = useTheme();
-  const controlBarStyle = controlBarStyles[layout ?? 'horizontal'];
+  const rtl = getRTL();
+  const controlBarClassName = useMemo(() => {
```

Comment:
`useMemo` is unnecessary here because `mergeStyles` returns a string (the className of registered CSS class).

Reply:
This is valid use I think because this will prevent mergeStyles from running again on component update unless a dependency changes. `mergeStyles` is an expensive method.